Sources/Variable.swift

```diff
@@ -50,6 +50,7 @@ public struct Variable: Equatable, Resolvable {
   public func resolve(_ context: Context) throws -> Any? {
     if (variable.hasPrefix("'") && variable.hasSuffix("'")) || (variable.hasPrefix("\"") && variable.hasSuffix("\"")) {
       // String literal
+      guard variable.count > 1 else { return "" }
```

Comment:
Move this logic to the if test:
```swift
if variable.count > 1 && (...) {
```

Reply:
@djbe Done! :)

Reply:
@djbe Fixed logic and tests and added changelog entry. Build passed 🟢